LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
```

Comment:
This is actually meant to be this way. It's telling the person to fill in their own copyright

Reply:
Fair point, removed the file from management